worldedit-core/src/main/java/com/sk89q/worldedit/regions/ConvexPolyhedralRegion.java

```diff
@@ -217,6 +219,26 @@ public boolean addVertex(BlockVector3 vertex) {
         return true;
     }
 
+    private boolean isCollinear(Set<BlockVector3> ab, BlockVector3 c) {
+        Iterator<BlockVector3> it = ab.iterator();
+        BlockVector3 a = it.next();
```

Comment:
It might make sense to give this a name more explicit to only working for vertex lists that only have two elements- otherwise it might lead to someone using this in the future expecting it to detect n-length collinearity. And then if that's done, it'd likely make sense to add a guard to the method to throw if a list of non-2 length is given